internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -399,25 +448,6 @@ func (provisioner *KopsProvisioner) UpgradeCluster(cluster *model.Cluster) error
 
 	logger.Info("Upgrading cluster")
```

Comment:
Doesn't this log message need to move since the stanza below it moved? Otherwise, unless I'm misunderstanding the code, the Provisioner will print "Upgrading cluster" after kops runs but before Terraform, which seems a little weird to me.

Reply:
It definitely does seem like that may be wrong, but I just double-checked and I think that is still pretty much the correct spot for it. The reasoning is that everything above that line is basically trying to get the kops manifests in the right state for what we want to "upgrade" to and then starting with the terraform plan+deploy we are making actual resource changes that affect the running cluster.